contracts/IssuanceFactory.sol

```diff
@@ -88,45 +88,47 @@ contract IssuanceFactory is CurrencyFactory {
 	/// @param _symbol string symbol of the token
 	/// @param _decimals uint8 ERC20 decimals of local currency
 	/// @param _totalSupply uint total supply of the local currency
+	/// @param _metadata string IPFS hash for the CC token data.
```

Comment:
again

Reply:
It can be everything, but I want to say that the convention is to save it to IPFS